doc/devel/release_guide.rst

```diff
@@ -43,18 +48,38 @@ We automatically extract GitHub issue, PRs, and authors from GitHub via the API:
 Review and commit changes.  Some issue/PR titles may not be valid rst (the most common issue is
 ``*`` which is interpreted as unclosed markup).
 
+.. note
+
+   Make sure you authenticate against the github API (either via
+   keyring or via putting an oauth token in :file:`~/.ghoauth`).  If you
+   do not you will get blocked by github for going over the API rate
+   limits.
+
 
 .. _release_chkdocs:
 
-Check Docs
-----------
+Check and updating Docs
+-----------------------
+
+Before tagging, update the "what's new" and API changes listings.
+
+ 1. copy the current content to a file in :file:`doc/users/prev_whats_new`
+ 2. merging all of the files in :file:`doc/users/next_whats_new/` into
+    :file:`doc/users/whats_new.rst` and deleting the individiual files
+ 3. comment out the next whats new glob at the top
+
+Similarly for the API changes
+
+ 1. copy the current api changes to a file is :file:`doc/api/prev_api_changes`
+ 2. merge all of the files in :file:`doc/api/next_api_changes/` into
+    :file:`doc//whats_new.rst`
+ 3. comment out the next API changes at the top.
 
-Before tagging, update the what's new listing in :file:`doc/users/whats_new.rst`
-by merging all files in :file:`doc/users/next_whats_new/` coherently. Also,
-temporarily comment out the include and toctree glob; re-instate these after a
-release. Finally, make sure that the docs build cleanly ::
+In both cases step 3 will have to be un-done right after the release.
 
-  make -Cdoc O=-n$(nproc) html latexpdf
+Finally, make sure that the docs build cleanly ::
+
+  make -Cdoc O-n=$(nproc) html latexpdf
```

Comment:
That doesn't look right; it's definitely `O=`.

Reply:
The new version runs, the old version gives me weird errors...

```
jupiter@00:01  ➤  make -Cdoc O=n-$(nproc) html latexpdf
make: Entering directory '/home/tcaswell/source/p/matplotlib/doc'
usage: __main__.py [OPTIONS] SOURCEDIR OUTPUTDIR [FILENAMES...]
__main__.py: error: unrecognized arguments: n-8
make: *** [Makefile:20: html] Error 2
make: Leaving directory '/home/tcaswell/source/p/matplotlib/doc'
```

vs

```
(py37) ✘  ~/source/p/matplotlib [doc_backports ✭|⚑ 8]
jupiter@00:01  ➤  make -Cdoc O-n=$(nproc) html latexpdf
make: Entering directory '/home/tcaswell/source/p/matplotlib/doc'
Running Sphinx v1.8.4
loading pickled environment... done
[autosummary] generating autosummary 
....
```

Reply:
The "old" version has a typo and should be `O=-n$(nproc)`; the new version passes a different variable and does nothing.

Reply:
Sorry, miss-type

```
(py37) ✘  ~/source/p/matplotlib [doc_release_notes ✭|⚑ 8]
jupiter@00:15  ➤  make -Cdoc O=-n$(nproc) html latexpdf
make: Entering directory '/home/tcaswell/source/p/matplotlib/doc'
usage: __main__.py [OPTIONS] SOURCEDIR OUTPUTDIR [FILENAMES...]
__main__.py: error: argument -n: ignored explicit argument '8'
make: *** [Makefile:30: html] Error 2
make: Leaving directory '/home/tcaswell/source/p/matplotlib/doc'
```

Reply:
Because it should be `-j` not `-n` https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-j

Reply:
Which of course breaks things now that it is actually working....

```
Warning, treated as error:
the sphinx_copybutton extension does not declare if it is safe for parallel reading, assuming it isn't - please ask the extension author to check and make it explicit
```

Reply:
Which is fixed on their master branch....

Reply:
Ah, okay, must have been a copy-paste bug from the `make.py` switch.

Reply:
copybutton did a new release for us (thank @choldgraf !)